x-pack/plugins/ml/server/lib/alerts/jobs_health_service.test.ts

```diff
@@ -248,7 +248,16 @@ describe('JobsHealthService', () => {
     expect(logger.warn).not.toHaveBeenCalled();
     expect(logger.debug).toHaveBeenCalledWith(`Performing health checks for job IDs: test_job_01`);
     expect(datafeedsService.getDatafeedByJobId).not.toHaveBeenCalled();
-    expect(executionResult).toEqual([]);
+    expect(executionResult).toEqual([
+      {
+        context: {
+          message: 'Jobs  contain no errors in the messages.',
```

Comment:
It could be a bit simpler.

```suggestion
          message: 'No errors in the job messages.',
```

Reply:
Updated in https://github.com/elastic/kibana/pull/132496/commits/3ce48d366d65fa32602f126e50b5cdf6696039dc